### DISCUSSION TITLE
My parameters is shown by clap-info but not with bitwig

### DISCUSSION THREAD

**Discussion by logos-maker** [free-audio/clap#122] [Q&A] [answered]
I am programming a CLAP plug in C code. And I have extended the plugin-template.c example so it also has parameters.
But only the first parameter shows up in Bitwig.
But if I use clap-info all parameters show up like...

"clap.params" : {
         "param-count" : 3,
         "param-info" : [
            {
               "id" : "0x0",
               "module" : "pitch/oscillator",
               "name" : "pitch",
               "values" : {
                  "current" : 0.0,
                  "default" : 0.0,
                  "max" : 1.0,
                  "min" : -1.0
               }
            },
            {
               "id" : "0x1",
               "module" : "cutoff/filter",
               "name" : "cutoff",
               "values" : {
                  "current" : 0.0,
                  "default" : 0.5,
                  "max" : 1.0,
                  "min" : 0.0
               }
            },
            {
               "id" : "0x2",
               "module" : "resonance/filter",
               "name" : "resonance",
               "values" : {
                  "current" : 0.0,
                  "default" : 0.5,
                  "max" : 1.0,
                  "min" : 0.0
               }
            }
         ]
      }

And I don't know what to do about it, any suggestion? Have anyone seen something like this before and have an idea what is about?
Apparently there is some different ideas about how the pluginformat should be interpreted.

**Comment by robbert-vdh** [comment, marked as answer]
Do your parameters have the `CLAP_PARAM_IS_AUTOMATABLE` flag set? Bitwig only shows automatable parameters in its generic UI.

**Reply by logos-maker** [reply]
Ok! Thanks for the help. It worked! Now I can see my parameters.
